src/Behavioral.Automation/Services/DriverService.cs

```diff
@@ -164,13 +165,15 @@ public void SwitchToLastWindow()
         }
 
         public void ResizeWindow(int Height, int Width)
-        { 
+        {
             Driver.Manage().Window.Size = new Size(Width, Height);
         }
 
         public string MakeScreenShot()
         {
-            var fileName = Regex.Replace(TestContext.CurrentContext.Test.Name, @"(\\|\"")", string.Empty) + ".png";
+            var fileName = new string(TestContext.CurrentContext.Test.Name
```

Comment:
Why not use this?)
https://docs.microsoft.com/en-us/dotnet/standard/base-types/how-to-strip-invalid-characters-from-a-string

Reply:
Well, similar things was used before and turns out we've missed some characters from it. So now, we're removing all characters which are inappropriate by using Path call